src/xdocs/config_blocks.xml

```diff
@@ -706,16 +708,17 @@ for (int i = 0; ; ) this.notify(); // OK
         </p>
         <source>
 &lt;module name=&quot;NeedBraces&quot;&gt;
-    &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_CASE, LITERAL_DEFAULT&quot;/&gt;
-    &lt;property name=&quot;allowSingleLineStatement&quot; value=&quot;true&quot;/&gt;
+  &lt;property name=&quot;tokens&quot; value=&quot;LITERAL_CASE, LITERAL_DEFAULT&quot;/&gt;
+  &lt;property name=&quot;allowSingleLineStatement&quot; value=&quot;true&quot;/&gt;
 &lt;/module&gt;
         </source>
 
         <p>
           Next statements won't be violated by Check:
         </p>
         <source>
-switch (num) {
+switch (num)
+{
```

Comment:
return "{" back.
but please make inner content with indent as 2.

Reply:
@romani 
Does the indentation needs to be changed for Java code under source blocks such as given below on line 40

```
public void guessTheOutput()
{
    int whichIsWhich = 0;
    {
        int whichIsWhich = 2;
    }
    System.out.println("value = " + whichIsWhich);
}
```

Reply:
yes, please do.

Reply:
@romani Done